internal/util/vault.go

```diff
@@ -97,6 +109,7 @@ func InitVaultKMS(kmsID string, config, secrets map[string]string) (EncryptionKM
 	if !ok || kms.VaultRole == "" {
 		kms.VaultRole = vaultDefaultRole
 	}
+
```

Comment:
no need to change this here, place it in a cleanup commit if you think this is a useful change